src/models/lucky_cli_version.cr

```diff
@@ -1,15 +1,21 @@
+require "semantic_version"
+
 module LuckyCliVersion
   extend self
 
   def current_tag : String
     "v#{current_version}"
   end
 
-  def current_version : String
-    "0.26.0"
+  def current_version : SemanticVersion
+    SemanticVersion.new(0, 27, 0)
+  end
+
+  def min_compatible_crystal_version : SemanticVersion
+    SemanticVersion.new(0, 36, 1)
   end
 
-  def compatible_crystal_version : String
-    "0.36.1"
```

Comment:
I first put `">= 0.36.1", and that broke a few things since we sometimes put a "v" in front, and sometimes we don't. SemanticVersion is a bit more typesafe anyway 😂